tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -219,5 +248,37 @@ def handler(event: Dict[str, Union[str, int]],
     Raises:
         ExtractFilePathsError: An error occurred parsing the input.
     """
-    result = run_cumulus_task(task, event, context)
+
+    # Generating schema validators can take time, so do it once and reuse.
```

Comment:
This comment only applies if done outside a function. I'm guessing that the place this was copied from did it at the top of the file. We should do that here so we can reuse the schemas in tests.

Reply:
Concur, the loading of the schema and creation of the validator should be done outside the function so that it is loaded once on lambda load and not done for every call to the handler. This is done for performance only the validation would happen in the handler.

Reply:
fixed